docs/taichicon.rst

```diff
@@ -1,5 +1,5 @@
-TaichiCon
-=========
+Taichi Conferences
```

Comment:
I'm not sure about this change. I thought `TaichiCon` was a proper noun, e.g. "DockerCon 2020", not "Docker Conference". Better to double-check with @yuanming-hu